app/jobs/streams/twitch_webhook_registration_job.rb

```diff
@@ -0,0 +1,29 @@
+module Streams
+  class TwitchWebhookRegistrationJob < ApplicationJob
+    WEBHOOK_LEASE_SECONDS = 7.days.to_i
```

Comment:
The max is 10 days

I went with 7 so that we could kick off 2 bulk refreshes a week, and even if we missed one somehow, we would still not stop receiving webhooks